packages/fxa-auth-server/scripts/verification-reminders.js

```diff
@@ -20,7 +20,20 @@ const LIB_DIR = `${ROOT_DIR}/lib`;
 process.env.NODE_ENV = 'dev';
```

Comment:
Should this still be set to dev on a prod box?

Reply:
I think so, yep. There's nothing in the script that depends on `NODE_ENV` explicitly and I obviously don't expect anyone to run this in combination with `source`, in a way that could interfere with other processes.

Forcing it to `dev` means that it works in the integration tests, which run in a context where `NODE_ENV` has been forced to `production` regardless of the host setting.